app/routes/home.ts

```diff
@@ -49,14 +127,59 @@ export default class HomeRoute extends Route {
   @tracked plannedStartMax: any;
   @tracked keyword: any;
 
-  async model(params: object, transition: Transition<unknown>) {
+  @action
+  error(error: Error) {
+    let controller: any = this.controllerFor("home");
+    controller.set("errorMsg", error.message);
+    return true;
+  }
+
+  @action
+  loading(transition: any, originRoute: any) {
+    let controller: any = this.controllerFor("home");
+
+    controller.set("loading", true);
+    transition.promise.finally(() => {
+      controller.set("loading", false);
+    });
+  }
+
+  async model(params: any, transition: Transition<unknown>) {
+    // const model: any = this.modelFor("home");
+    // if (
+    //   model?.agendaItems?.toArray().length > 0 &&
+    // ) {
+    //   return model;
+    // }
+
+    const currentPage = 0;
+    const agendaItems = await this.store.query(
```

Comment:
We do not need to load agenda items to display the new Home page. 
![image](https://github.com/lblod/frontend-burgernabije-besluitendatabank/assets/3050307/ec124461-f087-419a-91d7-eb1cb2f91c05)